src/commands/LinkCommand.js

```diff
@@ -0,0 +1,30 @@
+import Command from "../Command";
+import PackageUtilities from "../PackageUtilities";
+
+export function handler(argv) {
+  return new LinkCommand(argv._, argv).run();
+}
+
+export const command = "link";
+
+export const describe = "Symlink together all packages which are dependencies of each other";
+
+export const builder = {};
+
+export default class LinkCommand extends Command {
+  get defaultOptions() {
```

Comment:
I believe this getter is not necessary, since you're not mixing in any defaults from the parent class.